test/tests/ver-1dc/ver-1dc.i

```diff
@@ -0,0 +1,280 @@
+cl = 3.1622e18
+N = 3.1622e22
+epsilon_1 = 100
+epsilon_2 = 500
+epsilon_3 = 800
+temperature = 1000
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 1000
```

Comment:
Please move this number to the top of the input file and use ${} to feed it here.